packages/extensions/openapi-to-cadl/test/complexObject/cadl-output/models.cadl

```diff
@@ -16,12 +16,14 @@ Unique identifier representing a specific product for a given latitude &
 longitude. For example, uberX in San Francisco will have a different product_id
 than uberX in Los Angeles.
 """)
+  @friendlyName("productId")
```

Comment:
I'm seeing an error locally when this decorator is added over model properties. Asking offline about it.